tests/nbo/input.dat

```diff
@@ -11,5 +11,5 @@ set basis cc-pvdz
 set r_convergence 12
 
 energy, wfn = energy('hf', return_wfn = True)
-NBOWriter(wfn).write('input.47')
+wfn.write_nbo('input.47')
```

Comment:
Do the nbo 47 or analysis files in the test need updating?

Reply:
Those used a spherical basis set, so the existing ones are fine. I'll update them to include a Cartesian example later this evening.